src/meetingreadinesschecker/DefaultMeetingReadinessChecker.ts

```diff
@@ -270,7 +270,21 @@ export default class DefaultMeetingReadinessChecker implements MeetingReadinessC
 
   async checkAudioConnectivity(audioInputDevice: Device): Promise<CheckAudioConnectivityFeedback> {
     let audioPresence = false;
+    const audioConnectivityMetrics = {
+      packetsReceived: 0,
+    };
     const audioVideo = this.meetingSession.audioVideo;
+
+    const checkAudioConnectivityMetricsObserver: AudioVideoObserver = {
+      metricsDidReceive(clientMetricReport: ClientMetricReport) {
+        clientMetricReport.getRTCStatsReport().forEach(report => {
+          if (report.type === 'inbound-rtp' && report.mediaType === 'audio') {
+            audioConnectivityMetrics.packetsReceived = report.packetsReceived;
```

Comment:
Nit: We can use += instead so that we don't fail the test based on a single report where there might be a disconnect due to network.

Reply:
this metric - report.packetsReceived - will give the latest packet received metric total count upto now. so, we will be safe.

Reply:
Yeah my bad, was confusing this with observable metrics. I can see you're accessing the raw report so we're good!